rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -277,12 +277,7 @@ s32 cellCameraInit()
 	cellCamera.todo("cellCameraInit()");
 
 	// Start camera thread
-	const auto g_camera = fxm::make<camera_thread>("Camera Thread");
-
-	if (!g_camera)
-	{
-		return CELL_CAMERA_ERROR_ALREADY_INIT;
-	}
```

Comment:
Idk if moving this check down is correct for null handler. The games still need to call the functions in correct order